tools/netsh/ebpfnetsh.rc

```diff
@@ -69,6 +69,11 @@ BEGIN
 \n                  determine the program type and expected attach\
 \n                  type.  If not specified, the default is xdp.\
 \n      pinpath   - Optionally the name to pin the program to.\
+\n      interface - Optionally the friendly name or index of a \
+\n                  network interface to which an XDP program should be attached. \
+\n                  If the interface is not specified (or index is set to 0), \
+\n                  the XDP program is attached to all network interfaces.\
+\n                  This parameter is ignored for other program types.\
```

Comment:
I think it should instead result in failure if specified for other program types.

Reply:
Fixed.